pentasharp/Program.cs

```diff
@@ -32,6 +48,19 @@ public static void Main(string[] args)
 
             app.UseRouting();
 
+            app.UseSession();
+
+            //app.Use(async (context, next) =>
```

Comment:
why is this commented out?

Reply:
I added this to ensure that users who are not logged in would be redirected to the login page if they attempted to access other routes. However, I later commented it out because our documentation specifies that all users should have the ability to view the bus schedule and see available taxis. Reservations can only be made when the user is logged in. Therefore, I will remove this part because it is not needed.